docs/topics/install/docker.rst

```diff
@@ -68,17 +68,24 @@ on your host machine::
     cd addons-server
     # Download the containers
     docker-compose pull  # Can take a while depending on your internet bandwidth.
-    # Start up the containers
-    docker-compose up -d
     make initialize  # Answer yes, and create your superuser when asked.
-    # On Windows you can substitute `make initialize` for the command:
-    docker-compose exec web make initialize
+    # On Windows you can substitute `make initialize` by the following commands:
+    docker-compose run --rm --user olympia web make initialize
+    docker-compose up -d
```

Comment:
needs updating.
I did
```
docker-compose run --rm --user 9500:9500 web make update_deps
docker-compose up -d
docker-compose exec --user olympia web make initialize
```
but maybe `--user 9500:9500` can be substituted for `--user olympia` too?

Reply:
Yeah in your case `olympia` should be `9500` (it's the default value, and on Windows we are not overriding that with the `uid` from the host)